File: backEnd/src/main/java/com/example/Atarefados/model/dto/TarefaEstadoDTO.java
package com.example.Atarefados.model.dto;

import com.example.Atarefados.model.EstadoTarefa;
import com.example.Atarefados.model.Tarefa;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@NoArgsConstructor
@AllArgsConstructor
public class TarefaEstadoDTO {

    private Tarefa tarefa;
    private EstadoTarefa estadoTarefa;
}

File: frontEnd/src/app/shared/models/dto/TarefaEstado.dto.ts
import {EstadoTarefaModel} from "../EstadoTarefa.model";
import {TarefaModel} from "../Tarefa.model";

export class TarefaEstadoDto {
  tarefa: TarefaModel;
  estadoTarefa: EstadoTarefaModel;
}
